src/pages/documents/[id].tsx

```diff
@@ -156,6 +175,36 @@ const DocumentPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
             </div>
           </section>
         )}
+
+        <div className="flex place-content-center">
+          <div className="mt-8 mx-auto max-w-screen-md px-4 lg:px-0">
+            <section className="mt-10" data-cy="top-documents">
```

Comment:
I believe this is the same block copied from the geo page, and we now have it in 3 places:
- Geo
- Document (family)
- Document (physical document)

Which leads me to believe we should be refactoring into its own managed component to keep it as DRY as possible.

Also I noticed (not your fault) that the `data-cy` attribute has been incorrectly copy pasta'd on the geo page (by me) and should be more indicative of the content too. Perhaps `data-cy="country-search"`